### ISSUE TITLE
wrong data for Japan

### ISSUE THREAD

**Issue by karpenko-eugene** [CSSEGISandData/COVID-19#33]
Wrong data for Japan in time_series_2019-ncov-Confirmed.csv on  2/5/20 23:00 - 2/6/20 9:00 - 2/6/20 14:20 (45 confirmed) because next value 25

**Comment by bjarkimg** [comment]
They where moved to the Cruse ship. but yes I agree. 
Current total is 26 for Japan

**Comment by HyperBI** [comment]
i also see some weir data in Japan: 6-2-2020 : 45 . After that it was around 25-28 and now on 16-2-2020 its 59.
Is this correct?

**Comment by bjarkimg** [comment]
yes, correct. But they should perhaps correct this spike.
![20200217_211736](https://user-images.githubusercontent.com/1289419/74686946-01491880-51cb-11ea-88be-275b5d778002.jpg)

**Comment by y26805** [comment]
Yes I feel this should be corrected, as it is potentially misleading.

[This article from the Japan Times ](https://www.japantimes.co.jp/news/2020/02/07/national/tokyo-olympic-organizers-worried-coronavirus/) suggests that there were 25 cases (86 cases subtracted by 61 cases on the cruise) as of 7th Feb (Fri), so the figure 25 is correct.
